configstack/running_limit.go

```diff
@@ -7,15 +7,20 @@ import (
 	"github.com/fatih/color"
 )
 
+const waitTimeBetweenThread = 500
```

Comment:
You're using the raw `500` value here. I think that it would be clearer to set it to `500 * time.Millisecond`. This way, when someone reads this, they know exactly the "unit" this value is in.